root/Readme.ja.md

```diff
@@ -0,0 +1,257 @@
+﻿# Open 棟梁
+Open 棟梁は、.NET Framework をベースとしたアプリケーション フレームワークです。  
+Open 棟梁のプログラムは、以下のリポジトリで公開しています。
+- OpenTouryoTemplates リポジトリ
+    - 概要  
+    Open 棟梁を使用したアプリケーションの**開発基盤 (テンプレート)** となるもの  
+    SQL Server を既定の DBMS とし、Visual Studio のバージョンごとにフォルダが分かれている
+    - 想定利用者  
+    Open 棟梁を使用してアプリケーションを開発する人
+- OpenTouryo リポジトリ (**現在のリポジトリ**)
+    - 概要  
+    OpenTouryoTemplates リポジトリの**母体**となるもの  
+    (Open 棟梁の各機能は、まず OpenTouryo リポジトリで開発され、その後 OpenTouryoTemplates リポジトリに取り込まれる)
+    - 想定利用者  
+    OSS 開発者
+
+このため、システム開発プロジェクトで Open 棟梁を使用される方は、[OpenTouryoTemplates リポジトリ](https://github.com/OpenTouryoProject/OpenTouryoTemplates)を使用してください。  
+以下、**OSS 開発者向け**に、Open 棟梁の利用方法をご紹介します。  
+
+このファイルの英語版は[こちら](Readme.md)から。
+
+## サンプルの実行手順
+Open 棟梁に同梱されるサンプルアプリケーションの実行手順は以下のとおりです。
+
+### 前提ツールのインストール
+あらかじめ、Visual Studio 2010 と、Open 棟梁がサポートしている以下の DBMS をインストールしておいてください。
```

Comment:
@yusukemaegawa 

There was an error in my pointed out.
These libraries are required in order to pass the compile. But the case of need for installation of the DBMS is that when the fix of libraries are related to the DBMS. I think it is better to add to that effect to this sentence.